contrib/rouge.py

```diff
@@ -0,0 +1,399 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Google Inc.
```

Comment:
Checking against the [Apache licensing howto](https://www.apache.org/dev/licensing-howto.html#alv2-dep), this should be fine.  Google's code is Apache-licensed and there is no "notice" file in the [origin repository](https://github.com/google/seq2seq).

We can include this file unmodified under the Apache license without any further entries in our license or notice files.